### ISSUE TITLE
Include source data

### ISSUE THREAD

**Issue by titaniumbones** [edgi-govdata-archiving/fiscal#2]
While some of it may be confidential, it might make sense to maintain a CSV of our financial data here.  this will allow us to do more visualizations on the fly -- @shaqsingh or @Blackglade might be able to help with that, ofr instance.  

In general, it would be cool if we had at least an informal workflow for visualization, e.g.:

- hey, I want to know what that process looks like
- first I will turn it into tabular data
- then I will ask one of our visualization people to whip up something along the lines that they already have done

**Comment by dcwalk** [comment]
Yeah I think this might be a really nice case for understanding a workflow to visualize stuff! (so many possibilities :)

**Comment by shapironick** [comment]
Okie! here's the latest financial info in a csv. i redacted the names of individual donors https://www.dropbox.com/s/he3dl3by1c4kuc0/EDGI_Balance%2BSheet%2BDetail.csv?dl=0 would love to see some visuals better than the ones i made!

**Comment by shapironick** [comment]
that does not include the many in kind donations and the direct payment of costs like those that TRU pay. Can TRU also release their side of this data?

**Comment by Blackglade** [comment]
This looks interesting to work with... but having a bit of trouble trying to just understand what all this means. If someone wants to get on call with me and go through the balance sheet understanding it, I'd love to find a way to make this into a visualization.

**Comment by shapironick** [comment]
Hi @Blackglade do you want to DM me on EDGI slack and we can set up a time to work through this data? thanks!

**Comment by Blackglade** [comment]
Sure. Sounds good.
